code/datums/martial/plasma_fist.dm

```diff
@@ -52,9 +61,50 @@
 	D.visible_message("<span class='danger'>[A] hits [D] with THE PLASMA FIST TECHNIQUE!</span>", \
 					"<span class='userdanger'>You're suddenly hit with THE PLASMA FIST TECHNIQUE by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", null, A)
 	to_chat(A, "<span class='danger'>You hit [D] with THE PLASMA FIST TECHNIQUE!</span>")
+	var/turf/Dturf = get_turf(D)
 	D.gib()
 	log_combat(A, D, "gibbed (Plasma Fist)")
-	return
+	if(nobomb)
+		return
+	if(!isliving(target) || !target.mind)
```

Comment:
shouldn't this be D? where is target defined?

Reply:
:grimacing: